src/main/java/thaumicenergistics/common/integration/ModuleAe2.java

```diff
@@ -0,0 +1,13 @@
+package thaumicenergistics.common.integration;
+
+import appeng.api.AEApi;
+import thaumicenergistics.common.items.ItemCraftingAspect;
+
+public class ModuleAe2 {
+
+    private ModuleAe2() {}
+
+    static void init() {
+        AEApi.instance().registries().itemDisplay().blacklistItemDisplay(ItemCraftingAspect.class);
```

Comment:
It makes no sense to make an "integration module" for AE2 because ThE depends on AE2, just make the call directly (maybe wrap in try/catch for backwards compat if we intend t support our for outside of GTNH)

Reply:
is it possible to conditionally apply ASM? AEApi doesn't exist at the time of ASM initialization

Reply:
I think even if we will support ThE outside of GTNH, we may rely on GTNH branch of AE2

Reply:
Agree. I strongly suggest to specify the minimum version in the `@Mod` annotation in that case though.